packages/examples/relay.config.json

```diff
@@ -5,10 +5,8 @@
   "projects": {
     "examples": {
       "language": "typescript",
-      "schema": "src/schema.graphql",
-      "schemaExtensions": [
-        "src/clientExtensions.graphql"
-      ]
+      "schema": "data/schema.graphql",
+      "schemaExtensions": ["data/client-extensions"]
```

Comment:
This is the meat of the change

Reply:
How I love the bugs related to usage of different platforms <3 I assume this time it is Mac specific issue

Reply:
We need the changefile but otherwise LGTM